acct-user/vdr/vdr-0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="VDR (VideoDiskRecorder) service user"
+ACCT_USER_ID=452
+ACCT_USER_HOME=/var/lib/vdr
+ACCT_USER_HOME_OWNER=vdr:vdr
+# note: the first group name in the array will become the primary group
+ACCT_USER_GROUPS=( vdr audio cdrom video )
+IUSE="graphlcd legacy-homedir remote serial systemd"
+
+acct-user_add_deps
+
+pkg_setup() {
+	# if user wants to preserve his existing vdr installation, he can set
+	# USE=legacy-homedir
+	use legacy-homedir && ACCT_USER_HOME=/var/vdr
+
+	# media-plugins/vdr-graphlcd
+	use graphlcd && ACCT_USER_GROUPS+=( lp usb )
+
+	# media-plugins/vdr-remote, _only_ when systemd is installed
+	if use remote; then
+		use systemd && ACCT_USER_GROUPS+=( input ) || \
+			ewarn "\nuse-flag remote only applicable on systemd systems\n"
+	fi
+
+	# media-plugins/vdr-serial: add group to access /dev/ttyS*
+	# on systemd systems: add "dialout", non-systemd systems: add "uucp"
+	if use serial; then
+		use systemd && ACCT_USER_GROUPS+=( dialout ) || ACCT_USER_GROUPS+=( uucp )
+	fi
+}
+
+pkg_preinst() {
+	# if useflag legacy-homedir is _not_ set, check if user vdr exists and what his homedir is
+	if ! use legacy-homedir; then
+		local EXISTING_HOME=$(egethome vdr)
+		einfo "existing home: x${EXISTING_HOME}x"
+		if [[ "${EXISTING_HOME}" = "/var/vdr" ]]; then
+			ewarn "The user \"vdr\" exists on this system, his current home directory is \"/var/vdr\""
+			ewarn "You have two options to continue:"
+			ewarn " - set USE=legacy-homedir for ${CATEGORY}/${PN} and keep using /var/vdr"
+			ewarn " - move /var/vdr to /var/lib/vdr manually and repeat to install ${CATEGORY}/${PN}"
+			die "user action required"
```

Comment:
Well, and what do I, if I want to start a fresh install, but keep my old '/var/vdr' setup for backup?

Reply:
I understand your point, so I can only propose to rename `/var/vdr` to `/var/vdr.bak` or similar. I will add another "eawarn" for this.
Any other/better idea?